docs/user.md

```diff
@@ -901,6 +901,7 @@ spec:
   standby:
     standby_host: "acid-minimal-cluster.default"
     standby_port: "5433"
+    standby_primary_slot_name: "slot"
```

Comment:
Would be nice to have a little bit more description with extra example and not just add one more line. Readers might think that the field is required.

Reply:
done